5.4/cli-dev/xdebug.ini

```diff
@@ -0,0 +1,8 @@
+xdebug.remote_port = 9000
```

Comment:
This is different. In case of xdebug it is port that IDE is listening on. This is generally outside of this container and it works fine (I've been using it for a while now).